clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/provider/view/KubernetesV2JobProvider.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019 Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.provider.view;
+
+import com.google.common.collect.ImmutableList;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.caching.agent.KubernetesCacheDataConverter;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.caching.view.model.KubernetesV2Manifest;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesManifest;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.model.KubernetesV2JobStatus;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.security.KubernetesSelectorList;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.security.KubernetesV2Credentials;
+import com.netflix.spinnaker.clouddriver.model.JobProvider;
+import com.netflix.spinnaker.clouddriver.model.JobState;
+import com.netflix.spinnaker.clouddriver.model.Manifest;
+import com.netflix.spinnaker.clouddriver.model.ManifestProvider;
+import com.netflix.spinnaker.clouddriver.security.AccountCredentialsProvider;
+import io.kubernetes.client.models.V1DeleteOptions;
+import io.kubernetes.client.models.V1Job;
+import lombok.Data;
+import org.apache.commons.lang.NotImplementedException;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Data
```

Comment:
Are we using the getters/setters generated by this?  Looks like `@Getter` on the `platform` attribute is all we need, which would leak less of the implementation.

Reply:
I don't think so, actually. Will remove! Thanks.